### PR TITLE
Use templated getRange() member function to get range.

### PR DESCRIPTION
**Pull request by toregge**
@geirst : please review
@baldersheim : FYI

### CODE REVIEW

**Review by geirst — DISMISSED** [review]
👍

File: streamingvisitors/src/vespa/vsm/searcher/floatfieldsearcher.cpp [hunk 1]
```diff
@@ -39,10 +39,8 @@ void FloatFieldSearcherT<T>::prepare(search::streaming::QueryTermList& qtl,
     for (auto qt : qtl) {
     size_t sz(qt->termLen());
         if (sz) {
-            double low;
-            double high;
-            bool valid = qt->getAsDoubleTerm(low, high);
-            _floatTerm.push_back(FloatInfo(low, high, valid));
+            auto range = qt->getRange<T>();
+            _floatTerm.push_back(FloatInfo(range.low, range.high, range.valid));
```
Comment (baldersheim) [inline streamingvisitors/src/vespa/vsm/searcher/floatfieldsearcher.cpp:43]: emplace_back ?

File: streamingvisitors/src/vespa/vsm/searcher/intfieldsearcher.cpp [hunk 2]
```diff
@@ -29,10 +29,8 @@ void IntFieldSearcher::prepare(search::streaming::QueryTermList& qtl,
     for (auto qt : qtl) {
         size_t sz(qt->termLen());
         if (sz) {
-            int64_t low;
-            int64_t high;
-            bool valid = qt->getAsIntegerTerm(low, high);
-            _intTerm.push_back(IntInfo(low, high, valid));
+            auto range = qt->getRange<int64_t>();
+            _intTerm.push_back(IntInfo(range.low, range.high, range.valid));
```
Comment (baldersheim) [inline streamingvisitors/src/vespa/vsm/searcher/intfieldsearcher.cpp:33]: emplace_back

File: streamingvisitors/src/vespa/vsm/searcher/floatfieldsearcher.cpp (hunk 1 above)
Comment (toregge) [reply streamingvisitors/src/vespa/vsm/searcher/floatfieldsearcher.cpp]: Done.

**Review by geirst — APPROVED** [review]
👍

**Pull request merged**